### PR TITLE
First missing positive number

### PR DESCRIPTION
**Pull request by smv1999**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Implemented the "First Missing Positive Number" problem in CPP

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#855

### CODE REVIEW

File: C-Plus-Plus/other/First_Missing_Positive_Number.cpp
Comment (siddharth25pandey) [inline C-Plus-Plus/other/First_Missing_Positive_Number.cpp]: Instead of writing many headers file, just add **bits/stdc++.h**

File: C-Plus-Plus/other/First_Missing_Positive_Number.cpp
Comment (siddharth25pandey) [inline C-Plus-Plus/other/First_Missing_Positive_Number.cpp:49]: Display message before and after n

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/other/First_Missing_Positive_Number.cpp
Comment (smv1999) [reply C-Plus-Plus/other/First_Missing_Positive_Number.cpp:49]: Okay I will make the suggested changes.

File: C-Plus-Plus/other/First_Missing_Positive_Number.cpp
Comment (siddharth25pandey) [inline C-Plus-Plus/other/First_Missing_Positive_Number.cpp:49]: Display suitable message before & after of n
**NOTE:** It is different from Commenting, you have to print it.

File: C-Plus-Plus/other/First_Missing_Positive_Number.cpp
Comment (siddharth25pandey) [inline C-Plus-Plus/other/First_Missing_Positive_Number.cpp]: Display message before cout

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/other/First_Missing_Positive_Number.cpp
Comment (smv1999) [reply C-Plus-Plus/other/First_Missing_Positive_Number.cpp:49]: Could you please elaborate on that @siddharth25pandey ? I don't quite understand.

Comment (siddharth25pandey) [reply C-Plus-Plus/other/First_Missing_Positive_Number.cpp:49]: @smv1999 For your better understanding, I am providing you an example
**cout<<"Enter the value of n upto which we have to find the first positive missing number"<<endl;**
Similarly others as mentioned.

Comment (smv1999) [reply C-Plus-Plus/other/First_Missing_Positive_Number.cpp:49]: Okay got it! I have made the changes again! Please check if everything is alright!

**Review by Abhijit2505 — APPROVED** [review]
LGTM

**Review by siddharth25pandey — APPROVED** [review]
@smv1999 Good to go
Able to merge now
Thanks!

**Comment by smv1999** [comment]
@atarax665 This PR is ready to be merged and is waiting for your approval. Please look into this!

**Review by atarax665 — APPROVED** [review]
LGTM :)

**Comment by iamrajiv** [comment]
@all-contributors please add @smv1999 for code

**Pull request merged**
